web/src/css/custom.css

```diff
@@ -1364,3 +1364,8 @@ details summary:hover {
 .admonition.admonition-caution .admonition-heading h5 {
   color: var(--custom-color-orange-800) !important;
 }
+
+.theme-back-to-top-button {
+  width: 2.5rem !important;
+  height: 2.5rem !important;
+}
```

Comment:
<img width="226" alt="Screenshot 2022-05-25 at 2 25 24 PM" src="https://user-images.githubusercontent.com/8291514/170194507-aeecb257-7d46-43cc-915f-6b581315557a.png">
  
might need to add a line at the bottom.